src/components/mainMenu/mainMenu.css

```diff
@@ -182,6 +182,65 @@
   overflow: hidden;
 }
 
+.bottomIcon {
+  width: 14px;
+  height: 14px;
+  font-size: 14px;
+}
+
+.bottomTabs {
+  position: absolute;
+  bottom: 32px;
+
+  & .label {
+    color: var(--color-grayscale-medium);
+    font-family: var(--content-font);
+    font-size: 12px;
+    font-weight: var(--font-weight-bold);
+    line-height: 18px;
+
+    &.active {
+      background: linear-gradient(90deg, var(--color-white) 0%, #eaeff7 100%);
+      color: var(--color-grayscale-medium);
+
+      &::before {
+        display: none;
+      }
+    }
+
+    & span:first-child {
+      margin-right: 3px;
+      margin-top: 7px;
+    }
+
+    & span:last-child {
+      margin: 5px 0 0;
+    }
+
+    & > div {
+      display: flex;
+      flex-direction: row;
+      align-items: flex-start;
+      margin-left: 20px;
+    }
+
+    & img {
+      width: 25px;
+      height: 25px;
+      margin-right: 10px;
+    }
+  }
+}
+
+.bottomTab {
+  background: var(--menu-background-color);
+  box-shadow: none;
+  width: 100% !important;
+  max-width: 100% !important;
+  border: none !important;
+  padding: 7px 10px !important;
+}
```

Comment:
Do we need all these `!important`s here? 
Can the selector rather be made more specific by moving `.bottomTab` inside `.bottomTabs` block in this file?